File: CSharp/Tests/FreeCoffeeCupsTest.cs
using Xunit;

namespace CSharp.Tests
{
    public class FreeCoffeeCupsTest
    {
        [Theory]
        [InlineData(6, 7)]
        [InlineData(3, 3)]
        [InlineData(7, 8)]
        [InlineData(12, 14)]
        [InlineData(213, 248)]
        [InlineData(16, 18)]
        public void TotalCups_IntNumberOfCups_ReturnIntTotalCups(int n, int expected)
        {
            var actual = FreeCoffeeCups.TotalCups(n);

            Assert.Equal(expected, actual);
        }
    }
}

File: js/tests/freeCoffeeCups.test.js
const totalCups = require("../freeCoffeeCups");

const data = [
  [6, 7],
  [3, 3],
  [7, 8],
  [12, 14],
  [213, 248],
  [16, 18]
];

describe("totalCups", () => {
  describe("Return total number of cups for every 6 cups ordered", () => {
    test.each(data)("totalCups(%i) = %i", (n, expected) => {
      let actual = totalCups(n);

      expect(actual).toEqual(expected);
    });
  });
});
